apps/passport-client/components/screens/LoginScreens/CreatePasswordScreen.tsx

```diff
@@ -120,15 +120,11 @@ export function CreatePasswordScreen() {
           <H2>Choose a Password</H2>
           <Spacer h={24} />
           This password will be used to generate an encryption key that secures
-          your data. Save your password somewhere you'll be able to find it
-          later. If you skip this now, you will be asked to set a password on
-          adding your first PCD.
+          your data. Save your password somewhere you'll be able to find later.
```

Comment:
note that we will review during #562. I mainly cut this down to unblock potential bugs during DMS/ZMS testing this week, and if we do end up increasing the copy again, we'll have #865 ready